starlite/middleware/exceptions/debug_response.py

```diff
@@ -0,0 +1,121 @@
+import inspect
+from html import escape
+from pathlib import Path
+from traceback import format_exception
+from typing import TYPE_CHECKING, List
+
+from starlite.enums import MediaType
+from starlite.response import Response
+from starlite.status_codes import HTTP_500_INTERNAL_SERVER_ERROR
+
+if TYPE_CHECKING:
+    from starlite.connection import Request
+
+tpl_dir = Path(__file__).parent / "templates"
+
+
+def get_symbol_name(frame: inspect.FrameInfo) -> str:
```

Comment:
Please destructure rather than import the namespace. Thus `FrameInfo` instead of `inspect. FrameInfo`. Then you can actually do:

`"FrameInfo"` and move this to type checking import only.